frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -66,7 +66,9 @@ const DashboardDefaultView = ({ className }: { className?: string }) => {
   }
 
   const isEmpty = !dashboardHasCards || (dashboardHasCards && !tabHasCards);
-  const isFullHeight = isEditing || isSharing;
+
+  // Embedding SDK has parent containers that requires dashboard to be full height to avoid double scrollbars.
+  const isFullHeight = isEditing || isSharing || isEmbeddingSdk();
```

Comment:
See the PR description. Passing `className` to this component does not work because the `height: 1px` will override the height you pass in.